src/config/tsconfig.ts

```diff
@@ -38,7 +38,7 @@ const createTSConfig = (): string => {
       sourceMap: true,
       strict: false, // we should turn this on when data is ready :/
       target: 'es2018',
-      lib: ['es2019'],
+      lib: ['es2019', 'DOM', 'DOM.Iterable'],
```

Comment:
the default injected lib for --target ES6 is DOM,ES6,DOM.Iterable,ScriptHost. (ref: https://www.typescriptlang.org/docs/handbook/compiler-options.html)

so I'm ganna accept this change